### PR TITLE
companion to https://github.com/cylc/cylc-flow/pull/4521

### PR DESCRIPTION
**Pull request by wxtim**
https://github.com/cylc/cylc-flow/pull/4521

### CODE REVIEW

File: src/user-guide/running-workflows/running-workflows.rst [hunk 1]
```diff
@@ -171,10 +171,33 @@ reload are not inserted into the pool automatically. The first instance of each
 must be inserted manually at the right cycle point, with ``cylc insert``.
 
 
+Files created at workflow start
```
Comment (hjoliver) [inline src/user-guide/running-workflows/running-workflows.rst:174]:
```suggestion
Files Created at Workflow Start
```

File: src/user-guide/running-workflows/running-workflows.rst [hunk 2]
```diff
@@ -171,10 +171,33 @@ reload are not inserted into the pool automatically. The first instance of each
...
+-------------------------------
+
+Configuration Logs
+^^^^^^^^^^^^^^^^^^
+
+At startup a folder ``log/flow-config`` is created where the flow configuration
+is recorded, with all templating expanded:
```
Comment (hjoliver) [inline src/user-guide/running-workflows/running-workflows.rst:181]:
```suggestion
At startup a folder ``log/flow-config`` is created to record the workflow configuration,
with all templating expanded:
```

File: src/user-guide/running-workflows/running-workflows.rst [hunk 3]
```diff
@@ -171,10 +171,33 @@ reload are not inserted into the pool automatically. The first instance of each
...
+
+- ``flow-processed.cylc`` - A record of the current workflow configuration
+  with templating expanded, but without being fully parsed: Duplicate sections
+  will not be merged.
```
Comment (hjoliver) [inline src/user-guide/running-workflows/running-workflows.rst:185]:
```suggestion
- ``flow-processed.cylc`` - the current workflow configuration
  with templating expanded, and duplicate sections unmerged.
```

File: src/user-guide/running-workflows/running-workflows.rst [hunk 4]
```diff
@@ -171,10 +171,33 @@ reload are not inserted into the pool automatically. The first instance of each
...
+- ``<datetime-stamp>-<start/restart/reload>`` - A record of the config at
+  the time a workflow was started, restarted or reloaded, parsed by Cylc:
+  Duplicate sections will be merged.
```
Comment (hjoliver) [inline src/user-guide/running-workflows/running-workflows.rst:188]:
```suggestion
- ``<datetime-stamp>-<start/restart/reload>`` - the config at each time the workflow
 was started, restarted, or reloaded; with duplicate sections merged.
```

File: src/user-guide/running-workflows/running-workflows.rst [hunk 5]
```diff
@@ -171,10 +171,33 @@ reload are not inserted into the pool automatically. The first instance of each
...
+
+.. note::
+
+   These are particularly useful files to look at if you are working with a
+   workflow definition containing many template variables as these are filled
+   in these files.
```
Comment (hjoliver) [inline src/user-guide/running-workflows/running-workflows.rst:194]:
```suggestion
   Look at these files to see the result of templating in the workflow config.
```

**Review by hjoliver — CHANGES_REQUESTED** [review]
:+1: just a couple of suggestions to make the text more concise.

**Review by datamel — APPROVED** [review]
I'm happy with the changes as suggested by @hjoliver.
I have built the docs and read through the changes.
Thanks @wxtim. Approving under the assumption that changes requested will be implemented.

**Review by hjoliver — APPROVED** [review]

**Comment by wxtim** [comment]
@hjoliver  I'm a little confused as to why you merged the doc pr before the associated change...

**Comment by MetRonnie** [comment]
I think we will have to temporarily revert this change as this is an rc1 change? And then revert the revert after the 8.0b3 docs have been deployed

**Comment by hjoliver** [comment]
Oops, sorry, bad :grimacing:

**Comment by hjoliver** [comment]
I've reverted this merge on master.  @wxtim has deleted this PR branch, but I've recovered the commits and will push to my fork to reopen.

**Pull request merged**